icu4c/source/data/misc/units.txt

```diff
@@ -1898,50 +1898,137 @@ units:table(nofallback){
         }
     }
     unitQuantities{
```

Comment:
I do not see a reason for just making new level without combining two or three entries together.

Reply:
Purpose: we need the unitQuantity elements in their original order.

Implementation: adding <FIFO> to the ldml2icu rules is the way to do this. All examples of FIFO that I found have /<FIFO>/ in the rule. If necessary, we could check with David if there's another way to do this, but I think this is normal.

Consequences: units.txt comes out like this. (This file is generated by the ldml2icu tool.)

Reply:
Yeah.  In ICU restext, `{}` is used for both an array and a dictionary!  This is a key difference from JSON, where `[]` is used for arrays instead.  What Hugo has written here is the correct way to make an array of dictionaries.